### ISSUE TITLE
Official MacOs 15.1, Can I Run this code?

### ISSUE THREAD

**Issue by MrAplha777** [CatMe0w/zouxian#15]
On official mac os 15.1 , can i run this service? Is there any problem with that? 
After Installing the service, can I enable SIP??

Please Answer!! Really important to me!

**Comment by MrAplha777** [comment]
I tried it. As soon as i disable SIP, zouxian service or code stop working.
 
 is there a way to work around that without disabling SIP?

**Comment by cyyself** [comment]
I'm using Version 15.1 (24B83), and there is no problem.

**Comment by MrAplha777** [comment]
yeah. But You have to disable SIP. is there a way to work around that without disabling SIP?
